src/main/ionic/peers/src/pages/my-profile/my-profile.html

```diff
@@ -24,38 +24,38 @@
     <ion-row>
       <ion-col class="card-container">
         <ion-row>
-          <ion-col text-center>
+          <ion-col text-center col-8 offset-2>
```

Comment:
You should use col-md-8/col-sm-8 and offset-md-2/offset-sm-2 so it will be "responsive".